src/main/kotlin/automaton/constructor/view/MainWindow.kt

```diff
@@ -154,6 +159,16 @@ class MainWindow(
                     testsController.createTests()
                 }
             }
+            menu("Algorithms") {
+                menu("Pushdown automaton") {
+                    item("Convert into context-free grammar").action {
```

Comment:
```suggestion
                    item("Convert to context-free grammar").action {
```
А вообще строки лучше из языковых файлов брать, такие файлы в проекте, кстати, есть